source/agora/consensus/data/genesis/Test.d

```diff
@@ -56,31 +56,42 @@ public immutable Block GenesisBlock = {
         signature:   Signature.init,
 
         enrollments: [
+            // NODE6
```

Comment:
The comment was deleted [here](https://github.com/bpfkorea/agora/commit/0b03ccfcabe3426142800263550e58e43523b234#diff-777ad51a9f1968fe0167b8879a60b890b67a5202d961561b03bd6a1c53592ea1) :/